zilliqa/src/pool.rs

```diff
@@ -127,6 +133,60 @@ impl TransactionPool {
         }
     }
 
+    pub fn preview_content(&self) -> TxPoolContent {
+        // First make a copy of 'ready' transactions
+        let mut ready: BinaryHeap<ReadyItem> = BinaryHeap::new();
+        for item in self.ready.iter() {
+            ready.push(*item);
+        }
+
+        let mut pending = Vec::new();
+        let mut pending_set = HashSet::new();
+
+        // Find all transactions that are pending for inclusion in the next block
+        while let Some(ReadyItem { tx_index, .. }) = ready.pop() {
+            // Should we care about nonceless txns?
```

Comment:
Oh yeah, that's annoying. Fair enough. I guess we should exclude them in that case.

Then can we at least update the two comments to explain that yes we are deliberately excluding them because they don't have a nonce?